website/docs/advanced/config-v2-migration-guide.mdx

```diff
@@ -50,12 +50,15 @@ In your application, replace the old ConfigCat SDK Key with the new one (i.e. wi
 
 ### Step 4: Deploy your application
 
-Deploy your application to production and wait until all your users upgrade to the new version. You can check the migration status on the ConfigCat Dashboard on the V2 config's page.
+Deploy your application to production and wait until all your users upgrade to the new version.
+You can check the migration status on the ConfigCat Dashboard [here](https://app.configcat.com/v2/migration-status).
```

Comment:
Szerintem nincs direkt link rá, és én amúgy azt mondom ,hogy nem is fontos, hogy legyen, mert egy ilyen link nagyon ritka esetben tudna egy ilyen direkt link betalálni a jó helyre (csak abban az esetben tudna jó helyre menni, ha a legutóbb kiválasztott config-od migráció alatt van). 
Én inkább elmondanám, hol tudja megnyitni ezt az oldalt (... menü a configra kattintva)